conbench/tests/api/test_benchmarks.py

```diff
@@ -774,6 +774,21 @@ def test_create_allow_just_sha(self, client):
         location = "http://localhost/api/benchmarks/%s/" % new_id
         self.assert_201_created(response, _expected_entity(benchmark_result), location)
 
+    @pytest.mark.parametrize("pr_number", [12345678, "12345678", None, "", "<absent>"])
```

Comment:
Love it!!

Reply:
What is supposed to happen if a floaty value is provided? Rejected (error), silently discarded, or transparently converted to int?